src/schedule/train_schedule.py

```diff
@@ -47,10 +48,11 @@ def __init__(
         self.platform_ids = platform_ids
         super().__init__(strategy, id_)
 
-    def _spawn(self, traci_wrapper: "ITraCiWrapper"):
+    def _spawn(self, traci_wrapper: "ITraCiWrapper", tick: int):
         """Spawns a train.
 
         :param traci_wrapper: The TraCi wrapper to give the train to.
+        :param tick: The current tick
         """
-        # Will be implemented in a later issue
-        raise NotImplementedError()
+        train = Train(f"{self.id}_{tick}", self.platform_ids, self.train_type)
```

Comment:
Would you mind adding the format to [this](https://github.com/BP2022-AP1/bp2022-ap1/wiki/Database-Management#identifier) wiki table?

Reply:
Done